code/frontend/src/components/Filtering/Fx.js

```diff
@@ -0,0 +1,185 @@
+/*
+REFERENCE DOWNLOADED FROM CODEPEN TO UNDERSTAND MECHANISM FOR SCROLLING FEATURE. DO NOT DELETE, I WILL DELETE WHEN I FINISH
+THANKS XX
+ */
+
+
+
+import React from 'react';
+import * as ReactDOM from 'react-dom';
+
+class FixedTable extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      items: props.items,
+      sortingProp: 'id',
+      sortingDirectionAsc: true
+    };
+
+    this.scrollSidebar = this.scrollSidebar.bind(this);
+    this.scrollBodyListHeader = this.scrollBodyListHeader.bind(this);
+  }
+
+  componentDidMount() {
+    var $fixedTableBodyList = ReactDOM.findDOMNode(this.refs.fixedTableBodyList);
```

Comment:
Try not to use `var`. Consider `let` or `const`.
Please avoid using `$` for variable names. Refer to this link: https://www.w3schools.com/js/js_conventions.asp

Reply:
Sure, that code is just an example to understand scrolling. I won't be using the same thing